### DISCUSSION TITLE
Absolute Noob Question - Simple Windows CMD Shell Command

### DISCUSSION THREAD

**Discussion by rolandwellinger** [vmware/govmomi#3592] [Community Help] [answered]
Hi All
Sorry for that absolute noob Question. But i do not understand how simply use govc in the cmd of windows:
```
C:\Users\r.wellinger\Downloads\govc_Windows_x86_64>govc host.info -host.ip=192.168.0.187
govc: specify an ESX or vCenter URL
```
I simply try to get info about my host. But i dont understand how to provide the correct URL or IP to the command.

Thanks for any help

**Comment by ithinkiborkedit** [comment, marked as answer]
-host in this case refers to a virtual host that resides on the ESX/VCenter cluster. Not the actual VCenter hostname/IP that you have set  in your GOVC_URL env variable.
